opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -707,13 +740,13 @@ namespace Opm
         }
 
 
-        bool isNewWell(const int w) const {
-            return is_new_well_[w];
+        bool effectiveEventsHappen(const int w) const {
+            return effective_events_happen_[w];
```

Comment:
Make sure this works if wells are shut or opened *during* a report step due to inoperable well or WTEST.

Reply:
Except this one, other comments were taken care of.